proto/keyserverconfig.proto

```diff
@@ -119,24 +119,114 @@ message KeyserverConfig {
 	// cluster configuration.
 	repeated Replica initial_replicas = 7;
 
-	// EmailProofToAddr specifies the additional allowed to address in email
-	// proofs. By default, only proofs sent to the user being registered all
-	// accepted. This option can be used to allow proofs emailed directly to
-	// the keyserver to be accepted (but the keyserver does NOT implement a
-	// SMTP interface right now).
+	// # Registration policy
+	// Specifying a registration policy inline in this configuration is
+	// DEPRECATED. Use RegistrationPolicy (@12) instead.  In
+	// versions that support the RegistrationPolicy, any value (except
+	// for the default) in there takes precedence over every inline
+	// configuration. No new configuration should be deployed with inline
+	// configuration set. 
+	// Those currently relying on this option should:
+	// 1) update keyserver to a version that supports @12
+	// 2) change the configuration file to specify @12
+	// 3) remove the value of this option from the configuration file
+
+	// EmailProofToAddr (DEPRECATED) specifies the additional allowed to
+	// address in email proofs. By default, only proofs sent to the user being
+	// registered all accepted. This option can be used to allow proofs emailed
+	// directly to the keyserver to be accepted (but the keyserver does NOT
+	// implement a SMTP interface right now).
 	string email_proof_to_addr = 8;
-	// EmailProofSubjectPrefix is used for DKIM-based email address
-	// registration.  The proof challenge is sent in the subject line, with an
-	// optional string preceding it. For example, if EmailProofSubjectPrefix =
-	// "account verification: ", then the proof email needs to have a subject
-	// line "account verification: abcd" for verify challenge abcd.
+	// EmailProofSubjectPrefix (DEPRECATED) is used for DKIM-based email
+	// address registration.  The proof challenge is sent in the subject line,
+	// with an optional string preceding it. For example, if
+	// EmailProofSubjectPrefix = "account verification: ", then the proof email
+	// needs to have a subject line "account verification: abcd" for verify
+	// challenge abcd.
 	string email_proof_subject_prefix = 9;
-
-	// EmailProofAllowedDomains specifies the domains for which this keyserver
-	// accepts email address registrations.
+	// EmailProofAllowedDomains (DEPRECATED) specifies the domains for which
+	// this keyserver accepts email address registrations.
 	repeated string email_proof_allowed_domains = 10;
 
 	bool insecure_skip_email_proof = 11;
```

Comment:
Sounds good -- no one else is depending on compatibility right now, so as long as you don't have anything depending on it internally, go ahead.